sdk/notificationhubs/notification-hubs/README.md

```diff
@@ -0,0 +1,91 @@
+# Azure Notification Hubs SDK for JavaScript
+
+Azure Notification Hubs provide a scaled-out push engine that enables you to send notifications to any platform (Apple, Android, Web, Windows, etc.) from any back-end (cloud or on-premises). Notification Hubs works great for both enterprise and consumer scenarios. Here are a few example scenarios:
+
+- Send breaking news notifications to millions with low latency.
+- Send location-based coupons to interested user segments.
+- Send event-related notifications to users or groups for media/sports/finance/gaming applications.
+- Push promotional contents to applications to engage and market to customers.
+- Notify users of enterprise events such as new messages and work items.
+- Send codes for multi-factor authentication.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/notification-hubs) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/notification-hubs) |
+[Product documentation](https://docs.microsoft.com/azure/azure-notification-hubs/) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/notificationhubs/notification-hubs/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge, and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Install the package
+
+```bash
+npm install @azure/notification-hubs
+```
+
+### Prerequisites
+
+- An [Azure Subscription](https://azure.microsoft.com)
+- An [App Notification Hubs](https://docs.microsoft.com/azure/notification-hubs/) resource.
+
+### Create an Azure Notification Hubs resource
+
+TBD
+
+### Authenticate the client
+
+AppConfigurationClient can authenticate using a [service principal](#authenticating-with-a-service-principal) or using a [connection string](#authenticating-with-a-connection-string).
```

Comment:
Still referring to AppConfigurationClient. Maybe find and update "configuration" in the folder?